### PR TITLE
Deleting container directory from persist/runx

### PR DESCRIPTION
**Pull request by zed-rishabh**
Signed-off-by: zed-rishabh <rgupta@zededa.com>

### CODE REVIEW

File: pkg/pillar/cmd/volumemgr/initialvolumestatus.go [hunk 1]
```diff
@@ -269,7 +269,7 @@ func gcResetObjectsLastUse(ctx *volumemgrContext, dirName string) {
 }
 
 func deleteFile(filelocation string) {
-	if err := os.Remove(filelocation); err != nil {
+	if err := os.RemoveAll(filelocation); err != nil {
```
Comment (deitch) [inline pkg/pillar/cmd/volumemgr/initialvolumestatus.go:272]: This couldn't hurt, so why not? But why is it necessary?

Comment (zed-rishabh) [reply pkg/pillar/cmd/volumemgr/initialvolumestatus.go:272]: Actually, we are iterating over ```persist/runx/pods/prepared``` directory at bootup for populating initial status of container instances. So, we need to delete stale/obsolete container directories from the persist.

Comment (deitch) [reply pkg/pillar/cmd/volumemgr/initialvolumestatus.go:272]: I was not aware this was called by anything else. In any case, as we said, it certainly cannot hurt!

Comment (zed-rishabh) [reply pkg/pillar/cmd/volumemgr/initialvolumestatus.go:272]: > I was not aware this was called by anything else.
For reference, https://github.com/lf-edge/eve/pull/970

**Pull request merged**